src/validator/validators.js

```diff
@@ -48,7 +48,7 @@ var _validatePattern = function _validatePattern(value, callback, options) {
 var validators = {
     numeric: {
```

Comment:
It seems that there is no unit test coverage of the `numeric` validator at all. Could we take this opportunity to add one?

Reply:
This was not planned in the 1sp, I extended the existing one to check for the numeric selector for the validator.